dir.targets

```diff
@@ -14,6 +14,7 @@
   <PropertyGroup>
```

Comment:
Can I assume this entire file is needed for BuildTools?

Reply:
Yep. This entire file can go away once the product is off of BuildTools. I found another thing still using it, so it is still here (but soon!)